System/Random.hs

```diff
@@ -240,62 +212,53 @@ mutableByteArrayContentsCompat :: MutableByteArray s -> Ptr Word8
 -- >>> import Control.Monad (replicateM)
 -- >>> import Data.Bits
 -- >>> import Data.Word
--- >>> import System.IO (IOMode(WriteMode), hPutStr, withBinaryFile)
 -- >>> :set -XFlexibleContexts
 -- >>> :set -fno-warn-missing-methods
 
 -- | The class 'RandomGen' provides a common interface to random number
 -- generators.
-{-# DEPRECATED next "Use genWord32[R] or genWord64[R]" #-}
-{-# DEPRECATED genRange "Use genWord32[R] or genWord64[R]" #-}
+{-# DEPRECATED next "No longer used" #-}
+{-# DEPRECATED genRange "No longer used" #-}
 class RandomGen g where
-  {-# MINIMAL (next,genRange)|((genWord32|genWord32R),(genWord64|genWord64R)) #-}
+  {-# MINIMAL split,(genWord32|genWord64|(next,genRange)) #-}
```

Comment:
@curiousleo Hopefully this minimal definition is simple enough for you ;)

Reply:
Certainly an improvement! :)